smoke/features/steps/steps.py

```diff
@@ -454,12 +437,53 @@ def del_pods(context):
             buildpods.append(i.metadata.name)
     for pod in buildpods:
         res = oc.delete('pod',pod,current_project)
+        print("Deleting: ",res)
+
+@then(u'We rsh into the master pod and check the jobs count')
+def getjobcount(context):
+    for jobnames,_ in buildconfigs.items():
+        exec_command = 'cat /var/lib/jenkins/jobs/'+current_project+'/jobs/'+current_project+'-'+jobnames+'/nextBuildNumber'
+        count = oc.exec_in_pod(jenkins_master_pod,exec_command)
+        buildconfigs[jobnames] = str(count)
+    print(buildconfigs)
+
+@when(u'We delete the jenkins master pod')
+def deletemaster(context):
+    master_pod = getmasterpod(current_project)
+    res = oc.delete("pods",master_pod,current_project)
```

Comment:
curious why you delete the master pod vs. scaling it down like in https://github.com/openshift/origin/blob/master/test/extended/builds/pipeline_jenkins_e2e.go#L261

Reply:
not saying the pod delete, let the deployment config function recreate pod, is a bad thing per se

but in theory in the future if we wanted to examine things while jenkins was down, using scale down and scale up would allow for that

Reply:
@gabemontero I was thinking of adding that along side some more test cases to test the performance & robustness of jenkins template based install.

Reply:
so you mean in a subsequent PR, where in addition to those perf and robustness tests you possibly change this test to use scale vs. delete ?

Reply:
Yes